### ISSUE TITLE
Add a node provider

### ISSUE THREAD

**Issue by daKmoR** [jspm/generator#21]
While working locally it would be nice to be able to use the local resources.

e.g. something like this

```js
import { Generator } from '@jspm/generator';

const generator = new Generator({
  /*
   * Default: 'jspm'
   * Supported: 'jspm', 'jspm.system', 'skypack', 'jsdelivr', 'unpkg', 'node'.
   */
  defaultProvider: 'node',
});

// Install a new package into the import map
await generator.install('lit-element');

console.log(JSON.stringify(generator.getMap(), null, 2));
/*
 * Outputs the import map: 
 * {
 *    "imports": {
 *      "lit-element": "/node_modules/lit-element/lit-element.js",
 *      "lit-html": "/node_modules/lit-html/lit-html.js",
 *      "@lit/reactive-element": "/node_modules/@lit/reactive-element/reactive-element.js",
 *      "@lit/reactive-element/decorators/": "/node_modules/@lit/reactive-element/decorators/"
 *    }
 * }
 */
 
```

what do you think?

--
PS: adjusted request to use the name "node" instead of "local"

**Comment by guybedford** [comment]
I like the concept - would it make sense to call it a "node" or "node_modules" provider?

**Comment by daKmoR** [comment]
`node` sounds good... as it basically saves the  result of the node resolution into an import map

more or less  ☺️

**Comment by guybedford** [comment]
Released in 1.0.0-beta.4.
